newIDE/app/src/MainFrame/EditorContainers/HomePage/BuildSection/index.js

```diff
@@ -236,168 +288,173 @@ const BuildSection = ({
-                  <Column expand>
-                    <Text color="secondary">
-                      <Trans>Location</Trans>
-                    </Text>
-                  </Column>
-                  <Column expand>
-                    <Text color="secondary">
-                      <Trans>Last edited</Trans>
-                    </Text>
-                  </Column>
-                </Line>
-              )}
-              <List>
-                {authenticatedUser.loginState === 'loggingIn' &&
-                projectFiles.length === 0 ? ( // Only show skeleton on first load
-                  new Array(10).fill(0).map((_, index) => (
-                    <ListItem style={styles.listItem} key={`skeleton-${index}`}>
-                      <Line expand>
-                        <Column expand>
-                          <Skeleton
-                            variant="rect"
-                            height={skeletonLineHeight}
-                            style={styles.projectSkeleton}
-                          />
-                        </Column>
-                      </Line>
-                    </ListItem>
-                  ))
-                ) : projectFiles.length > 0 ? (
-                  projectFiles.map(file => (
-                    <ProjectFileListItem
-                      key={file.fileMetadata.fileIdentifier}
-                      file={file}
-                      currentFileMetadata={currentFileMetadata}
-                      storageProviders={storageProviders}
-                      isWindowWidthMediumOrLarger={!isMobile}
-                      onOpenRecentFile={onOpenRecentFile}
-                      lastModifiedInfo={
-                        lastModifiedInfoByProjectId[
-                          file.fileMetadata.fileIdentifier
-                        ]
-                      }
-                      onManageGame={onManageGame}
-                      canManageGame={canManageGame}
-                    />
-                  ))
-                ) : (
-                  <ListItem style={styles.listItem}>
-                    <Column expand>
-                      <Paper
-                        variant="outlined"
-                        background="dark"
-                        style={styles.noProjectsContainer}
-                      >
-                        <BackgroundText>
-                          <Trans>No projects yet.</Trans>
-                        </BackgroundText>
-                        <BackgroundText>
-                          <Trans>
-                            Create your first project using one of our templates
-                            or start from scratch.
-                          </Trans>
-                        </BackgroundText>
-                      </Paper>
-                    </Column>
-                  </ListItem>
-                )}
-              </List>
+              <Text size="section-title">
+                <Trans>Start with a template</Trans>
```

Comment:
is there a chance this section is almost invisible because too many projects are displayed on this screen?

Reply:
Yes, that's how the mockup has been designed.